src/pages/NewSignin.vue

```diff
@@ -362,6 +363,12 @@ export default {
 
         if (this.enable_popup) {
           this.$router.push(`/form/${this.userInformation["student_id"]}`);
+          UserAPI.postUserSessionActivity(
+            this.userInformation["student_id"],
+            "sign-in",
```

Comment:
Do we have a constants file? If not we should create it and move all these strings there so they are all in one place

Reply:
Do we need a separate file? It's just one string per component/flow.